src/main/java/reposense/parser/CsvParser.java

```diff
@@ -170,6 +170,11 @@ protected String getOrDefault(final CSVRecord record, int colNum, String default
                 .collect(Collectors.toList());
     }
 
+    protected List<String> getAsListOrDefault(final CSVRecord record, int colNum, List<String> defaultList) {
```

Comment:
Just a question, why you want to have a default list argument for this method?  Why cannot initialize the default list here in the method body? Since no other is call this method?

Reply:
I can indeed initialize the default list in the method body. Done.
I was originally thinking that this method may be re-used, but there may not be a need to predict future usages.